tox.ini

```diff
@@ -43,7 +43,7 @@ deps =
     devdeps: git+https://github.com/numpy/numpy
     devdeps: git+https://github.com/astropy/astropy.git#egg=astropy
     devdeps: git+https://github.com/asdf-format/asdf
-    devdeps: git+https://github.com/spacetelescope/stpipe
+#    devdeps: git+https://github.com/spacetelescope/stpipe
```

Comment:
Do you want this to remain in the file (and commented)?

Reply:
I put that back - so much for my playing.